pipeline_tools/pipelines/optimus/optimus.py

```diff
@@ -27,7 +27,9 @@ def create_optimus_input_tsv(uuid, version, dss_url):
     # Parse inputs from metadata
     print('Gathering fastq inputs')
     fastq_files = [
-        f for f in primary_bundle.files.values() if f.file_format == 'fastq.gz'
+        f
+        for f in primary_bundle.files.values()
+        if f.file_format in ('fastq.gz', 'fastq')
```

Comment:
The file format field name is actually `format`.

https://github.com/HumanCellAtlas/metadata-schema/blob/e9acc562153d3e9941b91cea67f7828d1497ceef/json_schema/core/file/file_core.json#L31

Not sure if calling it `file_format` here is based on the schema, but if so, it should be `f.format` for re-ingested data :)

Reply:
I see, I will change it to `format`.

The reason why it's still working is because we are using @hannes-ucsc 's great lib here which shields us from the changes (https://github.com/HumanCellAtlas/metadata-api/blob/11480262abe484facebee016ce7e63a366c8743a/src/humancellatlas/data/metadata/api.py#L596)

I just checked our metadata version, it uses v6.0.0, which seems to be using the `format`: https://schema.dev.data.humancellatlas.org/core/file/6.0.0/file_core

Reply:
That's so great that the library worked!

Reply:
After property renames we introduce a wrapper for backwards compatibility. The wrapper is deprecated  so some IDEs show a warning when you use it. https://github.com/HumanCellAtlas/metadata-api/blob/develop/src/humancellatlas/data/metadata/api.py#L611